extensions/amp-playbuzz/0.1/validator-amp-playbuzz.protoascii

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-playbuzz
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "SCRIPT"
+  spec_name: "amp-playbuzz extension .js script"
+  mandatory_parent: "HEAD"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-playbuzz"
+    dispatch_key: true
+  }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-playbuzz-(latest|0\\.1).js"
+  }
+  attrs: {
+    name: "type"
+    value_casei: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-playbuzz.html"
+}
+tags: {  # <amp-playbuzz>
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "AMP-PLAYBUZZ"
+  also_requires_tag: "amp-playbuzz extension .js script"
+  attrs: {
+    name: "src"
```

Comment:
all optional:

Do you want any constraints on this attribute value to help folks spot errors? Specifically, you could enforce that this is a URL (see https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/0.1/validator-amp-list.protoascii for an example).

Similarly, do you want to require any of the `data-` attributes shown in your examples? All data- attributes are allowed by default, but this means that small typos don't typically generate errors and can be subtle to catch. You can require a set here if that is your preference.